### PR TITLE
fix: devicetype ID for temperature-measurement

### PR DESCRIPTION
**Pull request by rochaferraz**
#### Problem
* No commissioning QR code link shown on logs from temperature-measurement app
* Device ID show on second endpoint is not of temperature-measurement device type

#### Change overview
* QR is printed
* Value of temperature is set to 20C at startup
* Device ID changed on ZAP file to temperature-measurement

#### Testing
* Tested on ESP32 M5 unit

### CODE REVIEW

File: examples/temperature-measurement-app/esp32/main/main.cpp [hunk 1]
```diff
@@ -47,6 +56,24 @@ const char * TAG = "temperature-measurement-app";
...
+static void InitServer(intptr_t context)
+{
+    chip::Server::GetInstance().Init();
+
+    // Initialize device attestation config
+    SetDeviceAttestationCredentialsProvider(Examples::GetExampleDACProvider());
+
+    // Sets it to 20C
+    int16_t value_16     = 2000;
+    EmberAfStatus status = emberAfWriteAttribute(1, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_TEMP_MEASURED_VALUE_ATTRIBUTE_ID,
```
Comment (bzbarsky-apple) [inline examples/temperature-measurement-app/esp32/main/main.cpp:68]: Please use the type-safe attibute accessors.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]
Need to use type-safe accessors.

**Comment by bzbarsky-apple** [comment]
Also, shouldn't this be a PR against master?

**Comment by rochaferraz** [comment]
Purpose of commit is mostly to test interoperability of this temperature sample in this branch. Master has time-format-localization-server cluster, which doesn't build/exists in this branch.

I can eventually push a similar PR to master too.

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@rochaferraz By default, we don't allow branches on the main repo outside of master. Can this work be done on a fork?

**Comment by rochaferraz** [comment]
Yup, no problem, thanks.

**Pull request closed without merging**